services/efs-guardian/src/simcore_service_efs_guardian/services/efs_manager.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from fastapi import FastAPI
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+
+
+@dataclass(frozen=True)
+class EfsManager:
+    app: FastAPI
+
+    _efs_mounted_path: str
+    _project_specific_data_base_directory: str
+
+    @classmethod
+    async def create(
+        cls,
+        app: FastAPI,
+        efs_mounted_path: str,
+        project_specific_data_base_directory: str,
```

Comment:
```suggestion
        project_specific_data_base_directory: Path,
```
is it a directory or just a name?

Reply:
Its a directory name in the root directory of the mounted file system